README.md

```diff
@@ -1,2 +1,59 @@
-# Hello-Microverse
-First Module exercise
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <br/>
+
+  <h3><b>Hello Microverse</b></h3>
+
+</div>
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [👥 Authors](#authors)
+- [📝 License](#license)
+
+# 📖 Hello Microverse <a name="about-project"></a>
+
+**[Hello Microverse]** is a project to master best practises in coding
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Main</summary>
+  <ul>
+    <li><a href="https://www.javascript.com/">Node.js</a></li>
+  </ul>
+</details>
```

Comment:
- [ ] kindly consider customizing this section to contain only the tech stacks you used in this project to avoid confusion to the people visiting your readme 
- [ ] Kindly consider making sure that every section provided in the readme template you were given is present also  in your readme to meet the below requirement 👇 
> All main headers from the [README template](https://github.com/microverseinc/readme-template) should be present in your  repo.